### DISCUSSION TITLE
Faster Colab setting

### DISCUSSION THREAD

**Discussion by rodwd** [TheLastBen/fast-stable-diffusion#2355] [Q&A]
Hi there, Colab had "premium GPU" earlier. That setting used approx 10-15 units per hour whereas my current setting uses approx 2 units per hour. I do currently use:
- GPU
- A100
Is a different setting faster? Or how do I change to that premium setting which was available in the past?

**Comment by TheLastBen** [comment]
google pro is too expensive, A100 is more about memory than speed, switch to runpod or paperspace
